timecard_settings.xml

```diff
@@ -1,28 +1,32 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <settings xsi:schemaLocation="http://maven.apache.org/SETTINGS/1.1.0 http://maven.apache.org/xsd/settings-1.1.0.xsd" xmlns="http://maven.apache.org/SETTINGS/1.1.0"
           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
     <servers>
         <server>
-            <id>callisto-jparest</id>
-            <configuration>
-                <httpHeaders>
-                    <property>
-                        <name>Authorization</name>
-                        <value>Bearer ${ARTIFACTORY_TOKEN}</value>
-                    </property>
-                </httpHeaders>
-            </configuration>
+            <id>github-packages</id>
+            <!-- Any username would work as long as it's not empty -->
+            <username>DummyUser</username>
+            <password>${env.GITHUB_TOKEN}</password>
         </server>
     </servers>
 
     <profiles>
         <profile>
-            <id>artifactory</id>
+            <id>github</id>
             <repositories>
                 <repository>
-                    <id>callisto-jparest</id>
-                    <name>Callisto Jparest Repo</name>
-                    <url>https://artifactory.digital.homeoffice.gov.uk/artifactory/callisto-jparest</url>
+                    <id>github-packages</id>
+                    <name>Callisto jparest</name>
+                    <url>https://maven.pkg.github.com/UKHomeOffice/callisto-jparest</url>
+                    <snapshots>
+                        <enabled>true</enabled>
+                    </snapshots>
+                </repository>
+                <repository>
+                    <id>github-packages</id>
+                    <name>Callisto kafka commons</name>
+                    <url>https://maven.pkg.github.com/UKHomeOffice/callisto-kafka-commons</url>
```

Comment:
The 2 <repository> element have the same id: github-packages. Ideally this should be a single top level maven repository and not multiple entries, one for each artefact (e.g one for callisto-kafka-commons, one for callisto-jparest, etc.) 
See example from maven website: https://maven.apache.org/settings.html#repositories
Have you tried using a single top level repository (maybe https://maven.pkg.github.com/UKHomeOffice?)